shunit2_test_helpers

```diff
@@ -226,6 +226,13 @@ _th_showOutput() {
   unset _th_return_ _th_stdout_ _th_stderr_
 }
 
+# Constants used to test file and directory assertions
+TH_EXISTING_DIRECTORY=$(exec 2>/dev/null;cd -- "$(dirname "$0")" || exit; unset PWD; /usr/bin/pwd || /bin/pwd || pwd)
```

Comment:
Could you instead create an explicit directory, rather than using shell to construct one? E.g. "${SHUNIT_TMP}/this_directory_exists".

This makes the test much easier to read, and when a future somebody sees an error with your explicit directory name, they can search for that directory name in the shunit2 code and find the problem much faster.